### DISCUSSION TITLE
使用 file: UploadFile = File(...) 做为接口参数,导致/docs 报错无法打开

### DISCUSSION THREAD

**Discussion by LiNoZues** [fastapi/fastapi#8802] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
#api.py
from fastapi import APIRouter,UploadFile, File, 
router = APIRouter(
    prefix='/api',
    tags=['api'],
)
@router.get('/test')
def test_g(user: str):
    print(process)
    return process[user]

@router.post("/uploadfile/")
async def create_upload_file(file: UploadFile = File(...)):
    return {"filename": file.filename}

# setting.py
from fastapi import FastAPI
from fastapi.staticfiles import StaticFiles
...
def get_application() -> FastAPI:

    application = FastAPI(title=PROJECT_NAME, debug=DEBUG, version=VERSION)

    origins = [
        "*"
    ]

    application.add_middleware(
        CORSMiddleware,
        allow_origins=origins,
        allow_credentials=True,
        allow_methods=["*"],
        allow_headers=["*"],
    )
    application.add_exception_handler(UnicornException, login_exception_handler)

    application.mount("/static", StaticFiles(directory="D:/Work/StaticFile"), name="static")

    return application


app = get_application()


# main.py
from settings import app
from api.routers import login, api,socket

app.include_router(login.router)
app.include_router(api.router)
app.include_router(socket.router)

if __name__ == '__main__':
    import uvicorn

    uvicorn.run("main:app", reload=True, port=8000)
```

### Description

访问127.0.0.1:8000/docs 报错
![image](https://user-images.githubusercontent.com/46307844/130598004-65ecd28f-d65e-4750-9fde-5d6d372c1fed.png)
只要把 /uploadfile/这个接口代码注释掉就可以正常访问 docs

### Operating System

Windows

### FastAPI Version

0.68.0

### Python Version

py 3.8

**Comment by LiNoZues** [comment]
![image](https://user-images.githubusercontent.com/46307844/130598360-f1dee0c6-4d3c-484b-ab47-d38435dd4805.png)
![image](https://user-images.githubusercontent.com/46307844/130598387-4cfb82f8-cacf-46a0-ba5a-760404f5090e.png)

**Comment by Mause** [comment]
What error is in your console?

**Comment by LiNoZues** [comment]
> What error is in your console?
Its keyerror 
![image](https://user-images.githubusercontent.com/46307844/130710122-8309e447-9b12-47d0-aa47-968ee5560227.png)

**Comment by LiNoZues** [comment]
In fastapi->utils.py ->  get_model_definitions()
![image](https://user-images.githubusercontent.com/46307844/130710425-3f417e77-eb52-4ee2-86e6-72d9355b5c8a.png)
![image](https://user-images.githubusercontent.com/46307844/130710436-f1b0cd73-9488-404b-be4b-1074e9c24a36.png)

They have different memory addresses

But in this way
```
from fastapi import FastAPI, File, UploadFile

app = FastAPI()


@app.post("/files/")
async def create_file(file: bytes = File(...)):
    return {"file_size": len(file)}


@app.post("/uploadfile/")

async def create_upload_file(file: UploadFile = File(...)):

    return {"filename": file.filename}



if __name__ == '__main__':
    import uvicorn

    uvicorn.run("test:app", reload=True, port=8000)
```

they have same id
![image](https://user-images.githubusercontent.com/46307844/130710709-4cc53127-4c41-47c1-9272-264dc50a5b4b.png)

are there any problem with my code

**Comment by Bahat159** [comment]
from fastapi import UploadFile

from the documentation tutorial, 500 Internal server error still persist, while working with UploadFile

`@app.post("/uploadfile/")`
`async def create_upload_file(file: UploadFile):`
         `return {"filename": file.filename, "file_content_type": file.content_type}`

Is there a fix or better way to solve this issue?
@LiNoZues
